shared/actions/chat.js

```diff
@@ -551,7 +552,7 @@ function * _incomingMessage (action: IncomingMessage): SagaGenerator<any, any> {
           })
 
           if (message.type === 'Attachment' && !message.previewPath && message.messageID) {
-            yield put(loadAttachment(conversationIDKey, message.messageID, true, tmpFile(message.filename)))
+            yield put(loadAttachment(conversationIDKey, message.messageID, true, false, tmpFile('preview-' + message.filename)))
```

Comment:
maybe we can use some metadata from the server to make a good name, like based on the hash or something. i'm worried about us redownloading stuff and not cleaning up the tmp folder and etc etc